.github/workflows/create-jira-issue.yml

```
@@ -1,11 +1,146 @@
 jobs:
-  create-issue:
-    name: Create Jira issue
+  # Job 1: 기존 Jira 이슈와 연결 (qa 라벨이 있는 경우)
+  link-existing-jira:
+    if: contains(github.event.issue.labels.*.name, '🛠️ qa')
+    name: Link Existing Jira Issue
+    runs-on: ubuntu-latest
+    steps:
+      - name: Extract Jira Key
+        id: extract
+        run: |
```

actionlint (1.7.8)

"github.event.issue.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)